src/classpath.ts

```diff
@@ -73,18 +73,20 @@ async function buildClasspath(storagePath: string, cwd: string, tool: string): P
       }
       cmd = `${mvnCmd} dependency:build-classpath -Dmdep.pathSeparator='${separator}' -Dmdep.outputFile=${classpathFilePath}`;
     }else if(tool === "gradle"){
-      const gradleCmd = await findGradleCmd(cwd);
+      const gradleCmd = await findGradleCmd();
       if (!gradleCmd) {
         return null;
       }
-      cmd = `${gradleCmd} classPath -PoutputFile=${classpathFilePath}`;
+      cmd = `${gradleCmd} --output-file=${classpathFilePath}`;
     }else{
       return null
     }
 
     try {
-      const result = await workspace.runCommand(cmd, cwd);
-      if (!result?.includes('BUILD SUCCESS')) {
+      await workspace.runCommand(cmd, cwd);
+      const errorCMD = IS_WINDOWS? "echo %errorlevel%": "echo $?"
```

Comment:
You might want to test out this `echo %errorlevel%` on a windows machine. would effect both the mvn command and the gradle command.

Reply:
got it from this page: https://stackoverflow.com/questions/334879/how-do-i-get-the-application-exit-code-from-a-windows-command-line/11476681#11476681

Reply:
What OS(s) have you tested on?

Reply:
im just on Linux (Arch)

Reply:
Just had a thought on this.
I'm not sure we can guarantee that the error code execution is going to be the error code of the previous command. 

Everything is async and within nodejs so we can't really tell where that error code is coming from unless we specifically checked the pid of the last command 

Better to do what you suggested and check for "BUILD SUCCESS"

Reply:
Yeah, I was wondering the same thing. I may be why I checked the message in the first place, but I can't remember. I just updated the gradle-classpath.zip in the comment above. When writing to a file, It will output "BUILD SUCCESS" as the last message if there's no error.

Reply:
Will you have time this week to make the update to go back to the check for "BUILD SUCCESS"? I think that's the last bit before it's ready to merge once I do some testing on Windows.

Reply:
Made the change, working for me as expected.

Reply:
I've tested on Windows and there are a few minor issues, but I think those we'll be easier to fix and test once these changes are on master.